README.md

```diff
@@ -8,3 +8,6 @@ Run `npm install` and `bower update` to get your dependencies ready. `./node_mod
 
 #### Usage
 Run `grunt` to fire it up, and make sure you have Mongo installed (`brew install mongo`) and that you've started the Mongo dæmon with `mongod`. Enjoy! 
+
+#### "Secure" Access
+Identical resources are available at `/collections` AND `/secure/collections`. POST/PUT/DELETE actions for `/secure` resources require an api key, which can be retrieved with a post call to `/secure/collections/users`. The post body should contain a JSON object with "username" and "email values and will return a JSON object with an "apiKey" value. This can be passed as query string "api_key" for access to restricted `/secure` resources. 
```

Comment:
Please add a note that this is toy-level auth, is completely insecure, and should never, ever be used for anything even remotely important.